website/source/docs/providers/azurerm/r/managed_image.html.markdown

```diff
@@ -0,0 +1,99 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_managed_image"
+sidebar_current: "docs-azurerm-resource-managed-image"
+description: |-
+  Create a Managed Image.
```

Comment:
How about "Create a custom virtual machine image that can be used to create virtual machines."